src/components/Routes.js

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import NotFound from './notfound/notFound';
 import Welcome from './welcome/welcome';
-import Privacy from './privacy/privacy';
+import Login from './login';
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={Welcome} />
-    <Route exact path="/privacy" component={Privacy} />
```

Comment:
nice work @habinezadalvan  however I do think privacy is one of the pages we are supposed to have on our mockup so it might be better if you let it there

Reply:
Okay